widget/edit.controller.js

```diff
@@ -4,7 +4,7 @@
     .module("cybersponse")
     .controller("editRecordCtaBlock100Ctrl", editRecordCtaBlock100Ctrl);
 
-    editRecordCtaBlock100Ctrl.$inject = [
+  editRecordCtaBlock100Ctrl.$inject = [
```

Comment:
Make it in One line like other widgets, like 
editRecordCtaBlock100Ctrl.$inject = ['$scope', '$uibModalInstance', 'config, 'appModulesService, 'Entity']
And also specify strings in single quotes at all places in js file.